src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-nested-ternary */
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+export const PeoplePage = () => {
+  const [peopleFromServer, setPeopleFromServer] = useState<Person[]>([]);
+  const [error, setError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getPeople()
+      .then((data) => setPeopleFromServer(data))
+      .then(() => setIsLoading(false))
+      .catch(() => {
+        setIsLoading(false);
```

Comment:
Instead of repeating two methods and method then to turn off the loader you could use finally method